neard/Cargo.toml

```diff
@@ -5,61 +5,41 @@ authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2018"
 default-run = "neard"
 
+[[bin]]
+path = "src/main.rs"
+name = "neard"
+
+[[bin]]
+path = "src/../src/main.rs"
+name = "near"
```

Comment:
Unrelated to the PR, but there's a chance we no-longer need this work-around. If that's the case, we can remove both bins.

Reply:
Good point, but I want this PR to keep things identical to the previous behavior wherever possible, so we don't spend time on unrelated discussions and don't revert the whole refactoring if we happen to decide to revert this change for some reason. Indeed, I plan to submit a PR dropping the old `near` binary.